src/swagger.ts

```diff
@@ -39,6 +39,8 @@ export const SwaggerDocs = (app: INestApplication) => {
   );
 
   const options = {
+    // Swagger introduces custom URLs for JSON export https://github.com/nestjs/swagger/pull/2273
```

Comment:
I don't think we need the comment here

Reply:
Have removed and added brief description to `README.md` under `Swagger` section.